tools/triage.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+# this should enforce as much of docs/TRIAGING.md as possible, run in top directory of xeps repo
+
+# usage: triage.sh $pr_title [$pr_commit] [$merge_base]
+# the last two will be HEAD and `git merge-base $pr_commit master` by default if not sent in
+
+# for testing you can:
+# git fetch origin +refs/pull/1254/merge
+# git checkout FETCH_HEAD
+# and only send in pr_title
+
+# test cases:
+# multiple xeps changed: git checkout 58cec1b6fb30ccc3b44c9dff34b94f53e958c1ac; ./tools/triage.sh 'XEP-0072, XEP-0096, XEP-0214: fix xmlns in sipub examples' 58cec1b6fb30ccc3b44c9dff34b94f53e958c1ac e4d3f721c2cf9fc7ed42c3428d098b2b3be3c8ed
+# protoxep: git fetch origin +refs/pull/1254/merge; git checkout FETCH_HEAD; ./tools/triage.sh "ProtoXEP: Stream Limits Advertisement"
+# one xep changed: git fetch origin +refs/pull/1249/merge; git checkout FETCH_HEAD; ./tools/triage.sh 'XEP-0444: bla'
+
+set -euo pipefail
+
+pr_title="$1"
+shift
+
+pr_commit=""
+if [ $# -ge 1 ]
+then
+    pr_commit="$1"
+    shift
+else
+    pr_commit="HEAD"
+fi
+
+merge_base=""
+if [ $# -ge 1 ]
+then
+    merge_base="$1"
+    shift
+else
+    merge_base="$(git merge-base "$pr_commit" master)"
+fi
+
+xpath() {
+    set -euo pipefail
+    xmllint --nonet --noent --xpath "$2" "$1" --nowarning --dtdvalid xep.dtd
+}
+
+files_changed="$(git --no-pager diff --name-only "$pr_commit" "$merge_base")"
+
+set +e
+protoxep="$(echo "$files_changed" | grep -E '^inbox/[^.]+.xml$')"
+num_protoxeps=$(echo "$protoxep" | grep -v '^$' | wc -l)
+set -e
+
+if [ $num_protoxeps -ge 1 ]
+then
+    # we are dealing with a protoxep here
+    echo "tag: protoxep"
+    if [ $num_protoxeps -ne 1 ]
+    then
+        echo "tag: split-pr"
+        echo "error: multiple protoxeps cannot be created in 1 PR"
+        exit 1
+    fi
+    expected_title="ProtoXEP: $(xpath "$protoxep" '/xep/header/title/text()')"
+    if [ "$pr_title" != "$expected_title" ]
+    then
+        echo "tag: bad-title"
+        echo "error: expected title '$expected_title' got title '$pr_title'"
+        exit 1
+    fi
```

Comment:
I wouldn't want to make this a hard error--that seems just superfluous bureaucracy to me. 

Enforcing the ProtoXEP prefix is useful (though labelling could replace that eventually), but let's not get too nitpicky about the overall title.

Reply:
This is just a warning now